wsd/TileDesc.hpp

```diff
@@ -525,6 +526,8 @@ class TileCombined
     int getHeight() const { return _height; }
     int getTileWidth() const { return _tileWidth; }
     int getTileHeight() const { return _tileHeight; }
+    /// Returns the combined-tile's AABBox, i.e. min-position + max-extend
+    Util::Rectangle toAABBox() const { return _aabbox; }
```

Comment:
Could return "const Util::Rectangle&" I guess and avoid a copy?

Reply:
thank you, yes ofc - will fix

Reply:
done here as discussed https://github.com/CollaboraOnline/online/pull/10232